Tests/JobsTests/Scheduler/SchedulerTests.swift

```diff
@@ -0,0 +1,371 @@
+import NIO
+@testable import Jobs
+import XCTest
+
+final class SchedulerTests: XCTestCase {
+    
+    #warning("TODO: - Uncomment after adding in recurrence rule")
```

Comment:
We should leave this out for now and pull from git later, that way we aren't merging commented code.

Reply:
Agreed, will remove